docs/src/main/asciidoc/Getting_Started_Guide.adoc

```diff
@@ -9,6 +9,7 @@ WildFly team;
 :doctype: book
 :icons: font
 :source-highlighter: coderay
+ifdef::env-github[:imagesdir: images/]
```

Comment:
Not really an issue for this PR, but I wonder if we need any of this `ifdef::env-github` stuff as we generate the documentation.

Reply:
@jamezp I'll conduct a test using this PR branch...

Reply:
@jamezp It does what I thought it does. It allows the images to be properly rendered when looking at the adoc pages in wildfly/wildfly github repo.

Compare section 1.4.4 of this:

https://github.com/bstansberry/wildfly/blob/doc_fixes/docs/src/main/asciidoc/Getting_Started_Guide.adoc

with the same from a branch I just created where I removed the ifdef from the page:

https://github.com/bstansberry/wildfly/blob/doc_fixes/docs/src/main/asciidoc/Getting_Started_Guide.adoc

The latter is missing the big graphic showing the WildFly welcome page.

Reply:
Ah that makes sense.